.idea/encodings.xml

```diff
@@ -7,6 +7,7 @@
     <file url="file://$PROJECT_DIR$/docker-java-core" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/docker-java-transport-jersey" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/docker-java-transport-netty" charset="UTF-8" />
+    <file url="file://$PROJECT_DIR$/docker-java-transport-okhttp" charset="UTF-8" />
```

Comment:
could you please revert this change? This file is going to be ignored and the change is unrelated to the PR anyways

Reply:
Also see: https://github.com/docker-java/docker-java/pull/1312

Reply:
Agree. My IDEA updates this file silently =(